src/helpers/accessiblity.ts

```diff
@@ -0,0 +1,52 @@
+import { StyleSheet } from 'react-native';
+import { ReactTestInstance } from 'react-test-renderer';
+
+export function isInaccessible(instance: ReactTestInstance | null): boolean {
+  if (!instance) {
```

Comment:
explicit checks are micro-faster
```suggestion
  if (instance === null) {
```